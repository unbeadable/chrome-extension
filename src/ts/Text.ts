export class Text {
  readonly element: HTMLElement = document.createElement("span");

  constructor(txtContent: string, styles: string) {
    this.element.setAttribute("class", styles);
    this.element.innerText = txtContent;
  }
}
