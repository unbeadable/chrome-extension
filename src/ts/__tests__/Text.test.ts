import { Text } from "../Text";

describe("Badge", () => {
  it("should match snapshot", () => {
    expect(new Text("someText", "someStyle").element).toMatchSnapshot();
  });
});
