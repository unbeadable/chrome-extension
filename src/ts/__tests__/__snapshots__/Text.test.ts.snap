// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should match snapshot 1`] = `
<span
  class="someStyle"
/>
`;
